FAQS.md

```diff
@@ -0,0 +1,54 @@
+**Markor - Frequently Asked Questions**
+
+**Markor - FAQ**
+
+This Frequently Asked Question will be continually updated to accommodate new information with respect to new updates and changes on the App and will be used inside the App to help users find answers to their questions.
+
+
+**What is Markor?**
+
+Markor is an Android application developed by Gregor Santner. It allows you to write, save and open files in markdown, plain and .txt formats. A note app developed for easy and improved note taking in markdown and .txt. 
+
+
+**What size is Markor and hope it isn't harmful to device?**
+
+Markor is a simple and lightweigh app that doesn't comsume much of your device space or RAM when installed, it is well diagonised for any harm it might cause your device and has been confirmed harm free.
+ 
+
+**Where can I download the Markor?**
+
+Markor can be downloaded from various app store like Google Play Store and on F-droid app store. 
+[Google Play Store](https://play.google.com/store/apps/details?id=net.gsantner.markor)
+[F-droid](https://f-droid.org/repo/net.gsantner.markor_20.apk)
+
+
+**Can I backup my notes from Markor?** 
+
+There isn't backup feature yet on the app, presently notes can only be saved and stored directly to a folder in the device's file manager.
+
+
+**Can I import my notes to the app?**
+
+Yes, saved notes can be imported to the app either to edit or save in other formats. Copy, Paste and Edit botton has been provided for easy editing.
+
+
+**Are there Markor versions for other OS besides Android?**
+
+Currently, Markor supports only Android and other OS version might be provided in the nearest future.
```

Comment:
No, but you can use any texteditor on any platform to edit documents that are created by Markor and vice-versa.